File: cls/file/file.go
package main


import "fmt"


type Person struct {
    Name string
    Age  int
}


func main() {
    person := Person {
        Name: "test",
        Age: 15,
    }

    fmt.Printf("%v\n", person)
    fmt.Printf("%+v\n", person)
    fmt.Printf("%#v\n", person)
}

File: cls/file/file.cpp
#include <fstream>
#include <iostream>

class Person
{
    public:
        char Name[64];
        int Age;
};


int main()
{
    ofstream ofs;
    ofs.open("test.txt", ios::out | ios::binary);

    Person p("张三, 18");
    ofs.write((const char *)&p, sizeof(Person));
    ofs.close();
}
